lch/lch.js

```diff
@@ -32,6 +32,17 @@ function LCH_to_sRGB_string(l, c, h, a = 100, forceInGamut = false) {
 	}).join(" ") + alpha_to_string(a) + ")";
 }
 
+function LCH_to_sRGB_hex_string(l, c, h, a = 100, forceInGamut = false) {
```

Comment:
Forcing into gamut should not be an option: hex outside of gamut is nonsensical, or even not possible to express (how would you represent negative coordinates?)

Reply:
I noticed this was producing values like `#-293f-17`, which makes no sense. Since we’re changing it to a single output, the warning message could always display the `rgb(0% 0% 0%)` syntax. I also think this doesn’t need to be its own function, and the `LCH_to_sRGB-string` function could take a new parameter `format` to support hex

Reply:
Do note that gamut mapping wouldn't *necessarily* turn an sRGB color with negative components to black, that depends on the values of all components. E.g. `rgb(-1% 100% -1%)` is green, not black. But you don't need to worry about gamut mapping, that's done by an existing part of the code.

I agree it could be a parameter for the sRGB function, though then you have the wart that for hex you need to ignore the `forceInGamut` parameter.